external-crates/move/crates/move-analyzer/tests/colon_colon_import.snap

```diff
@@ -123,4 +136,22 @@ Struct 'AnotherDepStruct'
     TARGET     : '(use Import::another_dep::AnotherDepStruct)'
     ADDITIONAL EDIT: 'use Import::another_dep::AnotherDepStruct;
 
+    '
+Struct 'SomeStruct'
+    TARGET     : '(use Import::Import::SomeStruct)'
+    ADDITIONAL EDIT: 'use Import::Import::SomeStruct;
+
+    '
+
+== import.move ========================================================
+-- test 0 -------------------
+use line: 10, use_col: 23
+Module 'Import'
+Module 'another_dep'
+Module 'colon_colon'
+Module 'dep'
+Struct 'SomeStruct'
```

Comment:
This is the other important part - module member on the list of suggestions in addition to modules themselves